lib/shared/permissions.js

```diff
@@ -73,6 +73,21 @@ exports.isElevated = () => {
 };
 
 /**
+ * @summary Quote a string
+ * @function
+ * @private
+ *
+ * @param {String} string - input string
+ * @returns {String} quoted string
```

Comment:
Minor nit, but could you update the `@summary` and `@returns` to clarify that it's double-quoting?
I guess for the sake of completion, (if you really wanted to) you could move this to `utils.js` and add `singleQuoteString` too ;-)